### ISSUE TITLE
SymfonyWorld

### ISSUE THREAD

**Issue by lsmith77** [symfony/diversity#117]
SymfonyWorld will be a virtual event what do we need to think of in addition/ in comparison to an in person event?

* [ ] no legal name requirement
* [ ] a11y of the conference platform
  * [ ] a11y of the website
  * [ ] hardware requirements
  * [ ] captions
  * [ ] bandwidth requirements
* [ ] pricing
  * [ ] can we adjust pricing by country?
  * [ ] diversity scholarship

**Comment by lsmith77** [comment]
Some general conference guides for accessiblity and diversity&inclusion.

It seems like no guide has yet really been updated to cover virtual events.

* https://docs.google.com/document/d/1PQVkafUUZUuVAEqJZ1aFrAy3ce7DYXYhj-Mf4QvxOiI/edit#heading=h.g4z3ggf5bqqc
* https://discover-cookbook.numfocus.org/
* https://github.com/DutchPHPConference/conference-diversity-and-inclusion/blob/master/Guideline.md

**Comment by lsmith77** [comment]
FYI current the planned platform for the event is https://hopin.to/

**Comment by xavismeh** [comment]
Program: display of schedule in different timezones or some kind of tool allowing to ease that conversion process

**Comment by javiereguiluz** [comment]
About "different price per country" ... in the European Union that's an illegal practice for non-physical products. See https://europa.eu/youreurope/citizens/consumers/unfair-treatment/unfair-pricing/index_en.htm

**Comment by lsmith77** [comment]
good point .. but differences in price can be achieved in different ways .. for example via discount codes which we could distribute somehow.

**Comment by javiereguiluz** [comment]
About "no legal name requirement" ... to attend the event you only need a SymfonyConnect account, which requires to have a name, but it doesn't require to use a real valid legal name (you can name yourself anything you want, real or not).

However, when purchasing the ticket, you must provide either your real personal legal name or the real legal name of your company, because of the legal requirements for invoices.

**Comment by fabpot** [comment]
Regarding the pricing, we have set a very low price (compared to other online conferences like Drupal or others) for the early bird to let as many people as possible register. In any case, it's impossible to check that someone is from country A or B. So, we found it simpler to just have a very low and fair price.

**Comment by lsmith77** [comment]
Price could still be too high for some. But maybe the issue for people who cannot afford the 59 Euro early bird is more that they don't have access to a credit card (that allows international payments) to begin with.

I raised the idea on the topic of buying the new book but we could promote the idea of founding user groups. Founders get a free Symfony elephant and a signed book in their chosen language. They in turn can then collect email addresses for people who cannot afford the ticket.

As for people than can afford the ticket but are unable to get access to a way to make the payment. Not sure if we could work out some process where we can help somehow via a bulk order. Then again it feels like too big a risk in terms of fraud. So maybe for those we also just provide free tickets using the above process.

In order to be eligible a user group would have to have meet at least 3 times until SymfonyWorld, sending a picture.

In terms of content they could simply have presenters go over individual chapters of the book. Maybe this will even boost even more translations?
